src/components/PreviewConcept/PreviewConcept.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { FC } from 'react';
+import styled from '@emotion/styled';
+import { spacing } from '@ndla/core';
+import { injectT, tType } from '@ndla/i18n';
+import {
+  NotionDialogContent,
+  NotionHeaderWithoutExitButton,
+  NotionDialogLicenses,
+  NotionDialogText,
+} from '@ndla/notion';
+import { Remarkable } from 'remarkable';
+import { Concept } from '../SlateEditor/editorTypes';
+import { getSrcSets } from '../../util/imageEditorUtil';
+
+const StyledBody = styled.div`
+  margin: 0 ${spacing.normal} ${spacing.small};
+  .c-tabs {
+    margin-left: 0;
+    margin-right: 0;
+  }
+  .react-tabs__tab-panel--selected {
+    animation-name: fadeInLeft;
+    animation-duration: 500ms;
+  }
+`;
+
+interface Props {
+  concept: Concept;
+}
+
+const PreviewConcept: FC<Props & tType> = ({ concept }) => {
```

Comment:
Du bruker ikkje t nokon plass, så då kan du ta bort importen.
```suggestion
const PreviewConcept: FC<Props> = ({ concept }) => {
```